text/0000-fabric-private-chaincode-lite.md

```diff
@@ -594,3 +639,19 @@ Typical signature verifications involve a PKI where the ultimate root of trust i
 Note on Terminology: The current feature naming scheme includes several elements that originated in Intel&reg; SGX.
 Today, terms like "Enclave" are widely accepted and used to refer to a generic TEE, rather than a specific technology.
 The project team currently participates in the new Confidential Computing Consortium which aims to promote standards for deployment of workloads across TEEs/Enclaves, and we intend to align with their terminology as it evolves.
+
+# Appendix
+[appendix]: #appendix
+
+## Encryption
+
+***TODO***
+ - write an appendix explaining what is data is encrypted, what cryto is used; which keys are shared; 
```

Comment:
what is encrypted we already do mention as well as the symmetric encryption algos used, the only thing missing is key-size and encryption of requests (e.g., the public key encryption) [no MAC should be necessary in FPC lite]  But i guess having an appendix section which explicitly lists all crypto algorithms used makes sense ..